.github/dependabot.yml

```diff
@@ -15,4 +15,5 @@ updates:
     schedule:
       interval: "weekly"
     allow:
+      - dependency-name: "cypress"
```

Comment:
Just for testing purposes, the final list of managed dependencies should be defined later